style.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  height: 6968px;
+  width: 375px;
+  left: 12142px;
+  top: -3480px;
+}
```

Comment:
Kindly make the width of the container responsive so it fits on a big screen

![res](https://user-images.githubusercontent.com/95412943/212058462-30cc56f8-f253-492a-b4df-7bf4521a8133.PNG)